src/collector/top_score_collector.rs

```diff
@@ -57,7 +57,7 @@ pub struct TopDocs(TopCollector<Score>);
 
 impl fmt::Debug for TopDocs {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "TopDocs({})", self.0.limit())
+        write!(f, "TopDocs({}, {})", self.0.limit(), self.0.offset())
```

Comment:
that will be confusing I think. Can we make it TopDocs(offset={},max_hits={}) or something like that?